.tekton/testing-farm.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: testing-farm
+spec:
+  description: Initiate testing-farm test given a list of container images
+  params:
+    - name: SNAPSHOT
+      description: A list of container images that should undergo testing
+    - name: GIT_URL
```

Comment:
One thing I find weird about this is we take parameters like this `GIT_URL`...

Reply:
Thank you for your comments, all of those are part of the original #116 (and eventually merged in #120) - I will close them as I am not familiar enough with the `tekton` framework to have useful input. Hope that is okay. Please just reopen them if not.